assets/src/edit-story/components/form/numeric.js

```diff
@@ -20,17 +20,17 @@
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { rgba } from 'polished';
-import { useState, useRef } from 'react';
-
+import { useCallback, useRef, useState } from 'react';
+import Big from 'big.js';
```

Comment:
Is this needed for this use case? Could we just use `BigInt` (with the necessary polyfills) instead?

Reply:
This was the only way I got it reliably working to +/- 0.1 or +/- 1 :\ how would you suggest to do it?

Reply:
Not sure what I was thinking. Let's keep it this way 🙂

Reply:
This might be a good time to bring up: the only float we had left was `lineHeight`. And the spec for it has also changed. It's now a `%` value and an integer. I could be mistaken, but I don't think we have any other floats left or coming up. If so, we can potentially drop anything related to floats.

Reply:
As it is now already implemented, I think it also does not hurt to keep it. Once we don't have any more floats, we can remove it. If we decide to have more floats, we'll be lucky and already have it implemented.